### PR TITLE
[librespeed] New Chart

### PR DESCRIPTION
**Pull request by dcplaya**
#### Special notes for your reviewer:
Simple chart to run a HTML5 based speedtest server. Has options for history as well.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/librespeed/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: 1.0.6-12
+description: Librespeed is a HTML5 webpage to test upload and download speeds
+name: librespeed
+version: 1.0.0
+keywords:
+  - Librespeed
```
Comment (billimek) [inline charts/librespeed/Chart.yaml:7]: This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

File: charts/librespeed/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+home: https://github.com/billimek/billimek-charts/tree/master/charts/librespeed
+icon: https://github.com/librespeed/speedtest/blob/master/.logo/logo3.png?raw=true
+sources:
+  - https://github.com/librespeed/speedtest
+  - https://hub.docker.com/r/linuxserver/librespeed
+  - https://github.com/billimek/billimek-charts/tree/master/charts/Librespeed
```
Comment (billimek) [inline charts/librespeed/Chart.yaml:13]: This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

File: charts/librespeed/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+HTML5 based speedtest with password protected history
+
+**This chart is not maintained by the Librespeed project and any issues with the chart should be raised [here](https://github.com/billimek/billimek-charts/issues/new)**
+
+## TL;DR;
+
+```shell
+helm repo add billimek https://billimek.com/billimek-charts/
+helm install billimek/Librespeed
```
Comment (billimek) [inline charts/librespeed/README.md:11]: This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

File: charts/librespeed/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+## Introduction
+
+This code is adopted from the [Linuxserver Librespeed docker image](https://hub.docker.com/r/linuxserver/librespeed) which runs the [Librespeed application](https://github.com/librespeed/speedtest)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```shell
+helm install --name my-release billimek/Librespeed
```
Comment (billimek) [inline charts/librespeed/README.md:23]: This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

File: charts/librespeed/README.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+| `affinity`                         | Affinity settings for pod assignment                                     | `{}`                      |
+| `podAnnotations`                   | Key-value pairs to add as pod annotations                                | `{}`                      |
+| `deploymentAnnotations`            | Key-value pairs to add as deployment annotations                         | `{}`                      |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```shell
+helm install --name my-release \
+  --set config.timezone="America/New_York" \
+    billimek/Librespeed
```
Comment (billimek) [inline charts/librespeed/README.md:87]: This is really minor, but can we make this instance lowercase (e.g. `librespeed`)

File: charts/librespeed/Chart.yaml (hunk 1 above)
Comment (dcplaya) [reply charts/librespeed/Chart.yaml]: All are resolved! That's what I get for using find/replace

**Pull request merged**